scripts/ci/publish

```diff
@@ -1,13 +1,7 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
-set -euo pipefail
-: ${SNAP_CI?This script is designed to be run within SnapCI. Aborting in a cowardly fashion.}
+cd "$(dirname "$0")"
+source _common.sh
 
-# Login to NPM
-npmrc_contents="//registry.npmjs.org/:_authToken=${NPM_TOKEN}"
-echo $npmrc_contents > "${HOME}/.npmrc"
-echo $npmrc_contents > "${SNAP_WORKING_DIR}/.npmrc"
-
-# Publish
-npm install autopublish
-node_modules/.bin/autopublish .
+cd "../.."
+pygo publish
```

Comment:
One of the goals of `pygo` was to eliminate the pattern of certain scripts being run on CI (usually in a /scripts/ci/) folder. Instead `pygo` commands would be called directly from Jenkins and replicating them locally would be more obvious. One simplification to consider is to call `pygo publish` directly in the publish stage of this project's Jenkinsfile. Again, more of a preference than something I'm willing to block a PR merge on.